documentation-site/routes.js

```diff
@@ -58,9 +58,13 @@ const routes = [
         path: '/theming/theming-values',
       },
       {
-        text: 'Overrides',
+        text: 'Overriding components',
```

Comment:
This makes the whole sidebar wider. Also, overrides can override `styles` / `props` only?